package.json

```diff
@@ -8,7 +8,10 @@
     "cy:open": "cypress open"
   },
   "devDependencies": {
-    "@bahmutov/cy-api": "1.5.0",
-    "cypress": "7.2.0"
+    "@bahmutov/cy-api": "^1.6.2",
+    "cypress": "^9.3.1"
+  },
```

Comment:
Lets avoid the package imports here and use the fixture test account. We can create , update, and delete the dummy user account.

Reply:
done